chart/templates/daemonset.yaml

```
@@ -92,5 +92,13 @@ spec:
             {{- end }}
           resources:
             {{- toYaml $.Values.resources | nindent 12 }}
+          {{- with .Values.extraVolumeMounts }}
+          extraVolumeMounts:
+            {{- toYaml . | nindent 12 }}
+          {{- end }}
       priorityClassName: system-cluster-critical
+      {{- with .Values.extraVolumes }}
+      extraVolumes:
```

```suggestion
      volumes:
```

Same in chart/templates/deployment.yaml

[Pod API Spec](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#volumes)